packages/react-aria-components/src/Table.tsx

```diff
@@ -1213,6 +1257,7 @@ export const Cell = /*#__PURE__*/ createLeafComponent('cell', (props: CellProps,
     <TD
       {...mergeProps(filterDOMProps(props as any), gridCellProps, focusProps, hoverProps)}
       {...renderProps}
+      colSpan={cell.colspan}
```

Comment:
it looks like in our table hooks we check for `isVirtualized` as a proxy for the elementType at the moment. let's move this down into the useGridCell hook

we may expand the support to actually checking elementType later and i'd prefer that all the logic is together

Reply:
Do you mean something like adding something like this: `colSpan: isVirtualized? undefined: node.colspan,`? One potential issue I see with that is that it might be semantically wrong if you add virtualization to a RAC implementation. But that might be an edge case, and you would still have the aria-colspan attribute.

Reply:
👍 I have moved it now.